dts/bindings/battery/fuel-gauge.yaml

```diff
@@ -0,0 +1,12 @@
+#
```

Comment:
File path for this still including battery, if the goal is fuel-gauge and charger under a power supply umbrella, then the path should match probably

Reply:
Whoops.